frontend/src/components/account/AccountPage.tsx

```diff
@@ -472,10 +451,20 @@ export default function AccountPage({
           <div className={classes.avatarContainer}>
             {account.badges?.length > 0 ? (
               <ProfileBadge badge={account.badges[0]}>
-                <Avatar {...avatarProps} />
+                <UserAvatar
+                  mode={"read"}
+                  isNarrowScreen={isSmallScreen}
```

Comment:
I'm not sure how much sense it makes to pass this as a prop because the component could also just retrieve the screensize itself. I'm not sure what the best practise is here. Same goes for other places with UserAvatar